File: module5/web-service/src/main/java/com/webservice/model/LoHang.java
package com.webservice.model;

import lombok.*;

import javax.persistence.*;

@Entity
@Builder
@Setter
@Getter
@NoArgsConstructor
@AllArgsConstructor
public class LoHang {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer idLoHang;
    private String ma;
    @ManyToOne
    private SanPham sanPham;
    private int soLuong;
    private String ngayNhapHang;
    private String ngaySanXuat;
    private String ngayHetHan;

}

File: CodeGym/module5/exame-project/src/app/model/lo-hang.ts
import {SanPham} from './san-pham';

export interface LoHang {
  idLoHang?: number;
  ma?: string;
  sanPham?: SanPham;
  soLuong?: number;
  ngayNhapHang?: string;
  ngaySanXuat?: string;
  ngayHetHan?: string;
  tenSanPham?: string;
  giaTien?: number;
  donVi?: string;
}
